cmd/boiler/main.go

```diff
@@ -34,3 +37,98 @@ func main() {
 	stop()
 	xos.ExitFromError(err)
 }
+
```

Comment:
Why did this dummy dburlopener get moved into boiler's main?

Reply:
I didn't want it to be an importable package. Probably should have just moved it to internal. Will change that